flutter/example/ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -21,6 +21,6 @@
 	<key>CFBundleVersion</key>
 	<string>1.0</string>
 	<key>MinimumOSVersion</key>
-	<string>11.0</string>
+	<string>12.0</string>
```

Comment:
Is this intentional? since we updated the min version to 12 in v8

Reply:
I will try again if building/ci works with 11

Reply:
Only wirks with v12. I created a new PR based on the V8 branch.